rfp-proposals/rfp-proposal-filscan.md

```diff
@@ -0,0 +1,141 @@
+ 
+FilScan is a customized blockchain engine for Filecoin with searchable and analyzed data that allows all the actors to explore the valuable information in a fast and easy way. It could be accessible by standard web browsers and responsive on mobile from anywhere in the world.
+ 
+The project will include the following sections: 
+ 
+ 1. Dashboard: Real-time Blockchain summary statistics, Search Bar(Keywords: Best Block, Peer ID and Wallet Address), Navigation Bar, Top 10 Tipset and Top-10 Message.
+ 
+ 2. Blockchain Page: Real-time and detailed Blockchain data with browse and retrieval function.
+ 
+ 3. Mining Page: Mining information which includes Miner State, Miner Index and Rank.
+ 
+ ![homepage design](https://github.com/ipfs-force-community/devgrants/blob/master/rfp-proposals/home.jpeg)
+ 
+ ## Deliverables
+ 
+ 1. The fully functional website is accessible via a https:// site with high-availability and works on mobile friendly. 
+ 
+ 2. The codebase is open-sourced and dual-licensed under MIT and APACHE2 licenses.
+ 
+ 3. Well-documented Deployment Document, API Instruction and User-Guide.
+ 
+ ## Development Roadmap
+ 
+ ### Milestone 1
+ 
+ 1. UI/UX Design: Dashboard, Blockchain Page and Mining Page
+ 
+ 2. Solution Design: Interface, Database, System Structure
+ 
+ 3. Design Doc
+ 
+ - 1 Full-stack, 1 Arch, 1 Product Owner
+ 
+ - 120hrs
+ 
+ ### Milestone 2
+ 
+ 1. Implementation:
+  Dashboard: Real-time Blockchain summary statistics, Search Bar(Keywords: Best Block, Peer ID and Wallet Address), Navigation Bar, Top 10 Tipset and Top-10 Message.
+ 
+    *The data to display includes and is not limited to the scope of Top Level in the RFP. API optimization and requirement details need further discussion.*
+ 
+    Tipset Detail Page(Node Type, Block Detail Page, Message Detail Page.
+ 
+    *The data to display includes and is not limited to the scope of Secondary Views in the RFP.*
+ 
+ 2. Documentation: Project Plan and User Guide
+ 
+ - 3 Dev, 0.5 PM
+ 
+ - 240hrs
+ 
+ ### Milestone 3
+ 
+ 1. Implementation: Mining Page includes Miner Page, Proven Power Metrics and Dynamic Dashboard.
+ 
+ 2. Documentation: Codebase.
+ 
+ - 3 Dev, 0.5 PM
+ 
+ - 240hrs
+ 
+ ## Total Budget Requested
+ 
+ Total Budget: $32,000.00
+ 
+ | Role | Rate/Hr | HC | Man-Hour | Man-Week | Price |
+ |-----|---------|----|-----------|-------------|------|
+ | Arch | $80 | 1 | 40| 1 | $3,200.00|
+ | PM | $60 | 1 | 80| 2| $4,800.00|
+ | PO | $50 | 1| 40 | 1 | $2,000.00 |
+ | UI-Dev | $50 | 1 | 120 | 3 | $6,000.00 |
+ | Back-Dev| $50 | 2 | 320 | 4 | $16,000.00 |
+ 
+ ## Maintenance and Upgrade Plans
+ 
+ 1. Based on actual Filecoin interface to update real time data of Retrieval and Token Market.
```

Comment:
Can you clarify whether you will be making these changes on your own (at no additional cost)?

Reply:
Of course we will provide the essential function which our user need most, provided filecoin API can support the data which function need.  If filecoin team support enough, it will not take additional cost.